datasets/gov/vat/seklu_laboratorijos.csv

```diff
@@ -0,0 +1,15 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vat/seklu_laboratorijos,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,,,,,,,,
+,,,,Laboratorija,,,eil_nr,,,4,open,,Įgaliotos sėklų kokybės tyrimo laboratorijos,
+,,,,,eil_nr,string,,,,4,open,,Eilės numeris,
+,,,,,laboratorija,string,,,,2,open,,Įgaliotosios laboratorijos pavadinimas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"", type: ""ref"", ref: ""/datasets/gov/rc/jar/iregistruoti/JuridinisAsmuo"")	",4,open,"spinta:205, manifest:1290",,
+,,,,,adresas,string,,,,1,open,,Įgaliotosios laboratorijos adresas,
+,,,,,,comment,type,,"update(base: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"", type: ""ref"", ref: ""/datasets/gov/rc/ar/adresotaskas/AdresoTaskas"")	",4,open,"spinta:205, manifest:1290",,
+,,,,,pazymejimo_nr,string,,,,2,open,,Pažymėjimo Nr.,
+,,,,,pazymejimo_data,date,D,,,4,open,,Pažymėjimo data,
+,,,,,pastabos,text,,,,4,open,,Pastabos,
```

Comment:
```suggestion
,,,,,pastabos,string,,,,4,open,,Pastabos,
```
https://github.com/atviriduomenys/spinta/issues/204
Atsižvelgiant į tai, kad `text` šiuo metu nepalaikomas, esame sutarę nurodyti `string` ir nemažinti brandos lygio.
Kai bus įgyvendintas šis funkcionalumas, matyt dalį `string` tipo duomenų reikės pakeisti į `text`.